page-changelog.php

```diff
@@ -73,6 +73,7 @@
 <li>[minor] Improve visual feedback when installing market apps - <a href="https://github.com/owncloud/market/issues/64">market/#64</a></li>
 <li>[minor] Don't display license key in config report - <a href="https://github.com/owncloud/configreport/issues/27">configreport/#27</a></li>
 </ul>
+Release notes: <a href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-1">Release notes</a></br>
```

Comment:
Is it here? what do you need a wordpress account for? https://staging.owncloud.org/changelog/#latest10.0
it looks fine here, at least :) link to the release notes worked.